### PR TITLE
Do not modify specialist-publisher directly

### PR DESCRIPTION
**Pull request by tatyree**
This was causing us problems with submodule management. The required
changes are now checked in on the tribunals branch on our fork of the
repo. With the exeception of the ruby version change, these should
probably all be removed before we send a pull-request to GDS.

### CODE REVIEW

File: run-specialist-publisher.sh [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
 
-cp ./specialist-publisher-publish-permissions.rake ./specialist-publisher/lib/tasks
-rm specialist-publisher/.ruby-version
 cd specialist-publisher/
 bundle
 bundle exec rake db:setup
 bundle exec rake dev:setup
 bundle exec rake permissions:grant
-rm ./lib/tasks/specialist-publisher-publish-permissions.rake
-perl -p -i -e "s/^.+publication_alert_exporter\,.*$//" app/observers/abstract_specialist_document_observers_registry.rb
```
Comment (robmckinnon) [inline run-specialist-publisher.sh]: If we remove this substitution, should we commit the amended `app/observers/abstract_specialist_document_observers_registry.rb` file as part of this PR?

File: specialist-publisher-publish-permissions.rake [hunk 2]
```diff
@@ -1,8 +0,0 @@
-namespace :permissions do
-  desc "Give publish permssions to default user"
-  task :grant => :environment do
-    u = User.first
-    u.permissions << 'gds_editor'
-    u.save!
-  end
-end
```
Comment (robmckinnon) [inline specialist-publisher-publish-permissions.rake]: Should we be deleting this file? We still use this task in the shell script.

File: run-specialist-publisher.sh (hunk 1 above)
Comment (robmckinnon) [inline run-specialist-publisher.sh]: Ok, I see we're adding this file to the submodule repo instead. So no problem. https://github.com/ministryofjustice/specialist-publisher/commits/tribunals

File: specialist-publisher [hunk 3]
```diff
@@ -1 +1 @@
-Subproject commit d086fd4cf6586c253fddad3f27ba1597bdf57db4
+Subproject commit a167161631eb8e2cd11c6af0d179963964255c99
```
Comment (robmckinnon) [inline specialist-publisher]: Looks like this sha is out-of-date, due to changes I pushed this morning. Latest sha is: 79215df53c85438e52b20cce71029a2e2349ea59

See: https://github.com/ministryofjustice/specialist-publisher/commits/tribunals

**Comment by tatyree** [comment]
Last checking brings it up the most current specialist-publisher commit.

**Pull request merged**